### PR TITLE
Add support for more Browserstack capabilities

### PR DESCRIPTION
**Pull request by oemekaogala**
@AndreyBelym 

**Problem:**
Right now, there is no support for the setting these Browserstack capabilities:

- name
- browserstack.geoLocation
- browserstack.customNetwork
- browserstack.networkProfile

**Solution:**
Create new environment variables that could be used to set these capabilities.

### CODE REVIEW

File: test/mocha/browserstack-capabilities-test.js [hunk 1]
```diff
@@ -3,27 +3,42 @@ var browserStackProvider = require('../../');
 
 
 describe('Browserstack capabilities', function () {
+    after(function () { 
+        delete process.env.BROWSERSTACK_TEST_RUN_NAME;
+        delete process.env.BROWSERSTACK_GEO_LOCATION;
+        delete process.env.BROWSERSTACK_CUSTOM_NETWORK;
+        delete process.env.BROWSERSTACK_NETWORK_PROFILE;
+    });
+
```
Comment (AlexKamaev) [inline test/mocha/browserstack-capabilities-test.js]: Why do we need this code? Do these variables affect the further test execution?

Comment (oemekaogala) [reply test/mocha/browserstack-capabilities-test.js]: Yes, they do.

Comment (AlexKamaev) [reply test/mocha/browserstack-capabilities-test.js]: You are right, they definitely do.
The `BROWSERSTACK_TEST_RUN_NAME` var goes to the `BrowserStack Dashboard` and `BROWSERSTACK_CUSTOM_NETWORK/BROWSERSTACK_NETWORK_PROFILE` emulates Network condition. 
Could you please explain why you delete the `BROWSERSTACK_GEO_LOCATION` variable? It seems that that var does not affect the further tests.

Comment (AlexKamaev) [reply test/mocha/browserstack-capabilities-test.js]: Looks like BrowserStack has a bug when I use the `geoLocation` capability with BrowserStack Automate. If I do not delete the `process.env.BROWSERSTACK_GEO_LOCATION` code, BrowserStack fails for some reason. So, this code is also required to run the tests.

Comment (oemekaogala) [reply test/mocha/browserstack-capabilities-test.js]: Thanks! It looks like this also addresses https://github.com/DevExpress/testcafe-browser-provider-browserstack/issues/49

**Pull request merged**